multinode_runfabtests.sh

```diff
@@ -69,12 +175,23 @@ if [ ${PROVIDER} == "efa" ]; then
         FABTESTS_OPTS+=" -C \"-P 0\" -s $gid_s -c $gid_c -t all"
     fi
 fi
+
 bash -c "$runfabtests_script ${FABTESTS_OPTS} ${PROVIDER} ${SERVER_IP} ${CLIENT_IP}"
 
 if [ ${PROVIDER} == "efa" ]; then
     # dgram_pingpong test has been excluded during installation
     # (in install-fabtests.sh), because it does not work with "-E" option.
     # So here we run it separately using "-b" option
     echo "Run fi_dgram_pingpong with out-of-band synchronization"
-    run_dgram_pingpong_with_b ${SERVER_IP} ${CLIENT_IP}
+    run_test_with_expected_ret ${SERVER_IP} ${CLIENT_IP} "${HOME}/libfabric/fabtests/install/bin/fi_dgram_pingpong" "PASS"
+
+    # Run fi_rdm_tagged_bw with fork when different environment variables are set.
+    echo "Run fi_rdm_tagged_bw with fork"
+    run_test_with_expected_ret ${SERVER_IP} ${CLIENT_IP} "${HOME}/libfabric/fabtests/install/bin/fi_rdm_tagged_bw -p efa -K -E" "FAIL"
```

Comment:
My understanding is this script will be run by another script with `set -e`. So when one process failed, the whole test will immediately fail. Can you verify if it is the case?

Reply:
Yeah. In the case of `set -e`. If the one process failed, the whole test will immediately fail.

Reply:
Please refer how other scripts handle this issue. For example, `wget_check.sh` will `set +e` if `set -e` has been called, and restore the option e at the end if necessary.

Reply:
Sure.